src/OaiPmh/HarvesterCommand.php

```diff
@@ -365,6 +370,12 @@ protected function execute(InputInterface $input, OutputInterface $output)
         }
 
         // All done.
+        if (isset($settings['stopAfter'])) {
+            $this->writeLine(
+                'stopAfter option set; '
+                . 'probably not all sources processed completely.'
```

Comment:
Maybe adjust to:

'all sources may not have been fully harvested.'

?

Reply:
I notice that you did not make this change. Was it overlooked, or do you disagree with it?

Reply:
sorry, just overlooked that.